index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>City Animals</title>
+    <link rel="stylesheet" href="./src/home/index.css">
+</head>
+<body>
+    <header>
+        <h1>City Animals</h1>
+        <h2>Live One Day as an Animal</h2>
+    </header>
+        <form id="user-sign-up">
```

Comment:
Feel free to add some white space between sections of the HTML elements for better readability.